api/config/index.js

```diff
@@ -7,29 +40,36 @@ const metadata = {
   connectionLimit: process.env.DB_POOL_SIZE,
 };
 const standbyMetadata = {
-  port: 3306,
   minimumIdle: 0,
 };
 
-module.exports = {
-  mysql0: {
+// everything below is to pull from process.json
+const configArry = [];
+
+// if npm run local, then just define what cli gives, otherwise use process.json
+if (!!process.env.DB_NODES) {
+  const dbs = JSON.parse(process.env.DB_NODES);
+  dbs.forEach(db => {
+    const thisObj = {
+      ...metadata,
+      host: db.host,
+      port: db.port || 3306,
+    };
+    if (!!db.isPrimary && db.isPrimary === true) {
```

Comment:
not needed in javascript, if `isPrimary` is set to `true`,  `!!db.isPrimary` will be truthy.